pkg/apis/resource/types.go

```diff
@@ -370,21 +370,17 @@ type ResourcePool struct {
 	ResourceSliceCount int64
 }
 
-const ResourceSliceMaxSharedCapacity = 128
+// Defines the maximum number of devices that can be specified in a single ResourceSlice.
```

Comment:
You can cleanup the PR and move all changes to types.go files into the first commit.